tests/telemetry/test_writer.py

```diff
@@ -380,19 +383,19 @@ def test_update_dependencies_event_when_disabled(telemetry_writer, test_agent_se
             assert event["request_type"] != "app-dependencies-loaded"
 
 
-@pytest.mark.skip(reason="FIXME: This test does not generate a dependencies event")
 def test_update_dependencies_event_not_stdlib(telemetry_writer, test_agent_session, mock_time):
     # Fetch modules to reset the state of seen modules
     modules.get_newly_imported_modules()
+    telemetry_writer.reset_queues()
+    assert telemetry_writer._imported_dependencies == {}
+    del sys.modules["httpretty"]
+    import httpretty  # noqa F401
 
-    import string
-
-    new_deps = [string.__name__]
-    telemetry_writer._app_dependencies_loaded_event(new_deps)
```

Comment:
So the fix for this test is to make the `app-dependencies-loaded` event get triggered indirectly via an actual import statement instead of explicitly like this?

Reply:
I believe so